code/src/components/messageForm.css

```diff
@@ -0,0 +1,141 @@
+.happy-form, .message {
+  background-color:#f3f1f1;
+  border: 2px solid black;
+  box-shadow: 4px 4px black;
+  width: 300px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px;
+  word-wrap: break-word;
+  margin: 20px;
+  font-family: 'Inconsolata', sans-serif;
+}
+
+.happy-form textarea {
+  width: 100%;
+  resize: none;
+  padding: 10px;
+  font-size: 16px;
+  box-sizing: border-box;
+  margin: 10px 0;
+}
+
+.happy-form h3 {
+  margin: 0;
+}
+
+.happy-form button {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 50px;
+  width: 70%;
+  background-color: #ffadad;
+  border: none;
+  font-size: 14px;
+  border-radius: 25px;
+  padding: 10px;
+  font-weight: 500;
+  transition: 0.2s all; 
+  box-shadow: 7px 6px 28px 1px rgba(0, 0, 0, 0.24);
+}
+
+.happy-form button:hover {
+  cursor: pointer;
+}
+
+.happy-form button:focus {
+  outline:0 !important;
```

Comment:
While the designer in me wants to do the same thing here (outline: 0), it's generally considered bad for accessibility purposes. 

I know accessibility isn't a requirement in this project, and if it's a conscious choice because of it, then ignore this comment. 

But in case you weren't aware of this, I just thought I'd let you know. See more here: http://www.outlinenone.com/